src/server/report/app.js

```diff
@@ -217,15 +229,18 @@ class App extends LitElement {
 			`;
 		};
 
+		const browserFilter = data.browsers.length > 1 ? html`
+			<fieldset>
+				<legend>Browsers</legend>
+				${ data.browsers.map(b => renderBrowser(b))}
+			</fieldset>` : nothing;
```

Comment:
Realized I was a bit eager with testing all 3 browsers and hadn't really been thinking about the more common case of just one. This hides the browser filter entirely if there's only 1 to filter on.